06-Rules/src/utils/AuthService.js

```diff
@@ -0,0 +1,100 @@
+import { EventEmitter } from 'events'
+import { isTokenExpired } from './jwtHelper'
+import Auth0Lock from 'auth0-lock'
+
+export default class AuthService extends EventEmitter {
+  constructor(clientId, domain) {
+    super()
+    this.domain = domain
+    // Configure Auth0
+    this.lock = new Auth0Lock(clientId, domain, {
+      additionalSignUpFields: [{
```

Comment:
@alemata what if we simplify this removing additionalSignUpFields and address field from profile details?